pkg/certificate/manager.go

```diff
@@ -7,24 +7,19 @@ import (
 	"sync"
 	"time"
 
-	"github.com/openservicemesh/osm/pkg/announcements"
+	"github.com/cskr/pubsub"
+	"github.com/rs/zerolog/log"
+
 	"github.com/openservicemesh/osm/pkg/constants"
 	"github.com/openservicemesh/osm/pkg/errcode"
-	"github.com/openservicemesh/osm/pkg/k8s/events"
-	"github.com/openservicemesh/osm/pkg/logger"
-	"github.com/openservicemesh/osm/pkg/messaging"
-)
-
-var (
-	log = logger.New("certificate")
```

Comment:
Can we keep this so we have more context in the log output?

Reply:
ya that was a mistake from the rebase, fixing